examples/scripts/txTransformTransfer.ts

```diff
@@ -1,8 +1,15 @@
-import { UserFactory } from "../../libs";
+import axios from "axios";
+import { Commitment, UserFactory } from "../../libs";
 import { createZkpKeys } from "../../libs";
 import { randomL2TokenAddress, randomSalt } from "../../libs";
 import { config } from "./appConfig";
 
+const makeBlock = async (timeoutms: number) => {
+  await new Promise((resolve) => setTimeout(resolve, timeoutms));
```

Comment:
Why waiting here, is this to make sure that the tx is in a block? If so, please add a comment.

Guess you could also make a util and place it in `libs/utils` (there is at least +1 instance of `makeBlock` in the `atomicTransaction` example).

Reply:
I have placed it in `/examples/scripts/utils/blocks.ts` because it's only for the examples and not as a feature for using externally.